File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1864_basico.js
import { question } from "readline-sync";

function main() {
    const FRASE = (`LIFE IS NOT A PROBLEM TO BE SOLVED`).split(``);
    const N = get_Number();
    let mensagem = ``;

    for(let i = 0; i < N; i++) {
        mensagem += `${FRASE[i]}`;

    }
    print(mensagem);
}

function get_Number() {
    return Number(question());
}

function print(texto) {
    return console.log(texto);
}

main();

File: Exercicios_ED_I/Bee/1864_basico.cpp
#include <stdio.h>
#include <iostream>
#include <string.h>
using namespace std;

main() {
    string mensagem = "LIFE IS NOT A PROBLEM TO BE SOLVED", nova_Mensagem = "";
    int i, n;
    cin >> n;

    for (i = 0; i < n; i++) {
        nova_Mensagem += mensagem[i];
    }
    cout << nova_Mensagem << endl;
}